keyboards/bm40rgb/info.json

```
@@ -0,0 +1,62 @@
+{
+    "keyboard_name": "BM40HSRGB",
+    "url": "",
+    "maintainer": "qmk",
+    "width": 12,
+    "height": 4,
+    "layouts": {
+        "LAYOUT": {
```

```suggestion
        "LAYOUT_planck_mit": {
```
Please also add the `ortho_4x12` here.